kubelet-log-config/kubelet-log-config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-log-config
+  namespace: default
+  labels:
+    k8s-app: kubelet-log-config
+spec:
+  selector:
+    matchLabels:
+      name: kubelet-log-config
+  updateStrategy:
+    type: RollingUpdate
```

Comment:
as long as there will be a logic that only modifies the kubelet config if it doesn't have those settings already, this doesn't matter

Reply:
ACK. It looks like all the other ones have this in this repo. We can just keep it here.